README.md

```diff
@@ -38,6 +38,45 @@ export default defineConfig({
 })
 ```
 
+### Systematically adding deployments
+
+You can pass in deployments that should already be added into the app (refresh the site after the initial load).
+
+**Make sure the projectNames are unique.**
```

Comment:
Why would the project name need to be unique across specified deployments? In most instances (including your example below) the `projectName` is the same. Let me know if I'm missing something!

Reply:
This was because I had a check to create a new project when the user refreshed the page

I need to make sure that I don't already have the project created